README.md

```diff
@@ -12,7 +12,7 @@ go get github.com/xperimental/nextcloud-exporter
 
 ## Client credentials
 
-To access the serverinfo API you will need the credentials of an admin user. It is recommended to create a separate user for that purpose.
+To access the serverinfo API you will need the credentials of an admin user. It is recommended to create a separate user and an [API token](https://docs.nextcloud.com/server/latest/developer_manual/client_apis/LoginFlow/index.html) for that purpose.
```

Comment:
I can not find a mention of "API token" on the linked page. Are you perhaps referring to "app passwords" instead? (https://docs.nextcloud.com/server/latest/developer_manual/client_apis/LoginFlow/index.html#converting-to-app-passwords)

---

I don't know if linking to the API documentation will benefit the users though. It would probably be better to explain how to create an "app password" to be used with the exporter through the UI.

Reply:
In some cases its named "apptoken". The UI is not scriptable and generates shorter app passwords/token. You can still use the UI afterwards to restrict filesystem access.